compiler/qsc_hir/src/hir.rs

```diff
@@ -215,10 +215,25 @@ impl Display for Res {
 pub struct Package {
     /// The items in the package.
     pub items: IndexMap<LocalItemId, Item>,
+    /// The top-level statements in the package.
+    pub stmts: Vec<Stmt>,
```

Comment:
Aha! As mentioned in the comment on the test file, top-level statments are in the package struct but not included in the display logic below.